src/modules/Elsa.Workflows.Core/Models/Activity.cs

```diff
@@ -143,10 +143,15 @@ protected async ValueTask CompleteAsync(ActivityExecutionContext context)
 
     async ValueTask IActivity.ExecuteAsync(ActivityExecutionContext context)
     {
+        // Invoke execution logging behavior prior to execution start.
+        var executionLoggingBehavior = Behaviors.First(x => x.GetType() == typeof(ExecutionLoggingBehavior));
```

Comment:
Can we add a "Priority" property to behaviors so that we can sort by them, instead of having hardcoded knowledge about the ExecutionLoggingBehavior?